ONSdigital/census-rm-case-processor
# Dates serialised in UTC format

# Motivation and Context
Dates are not serialised to JSON in UTC format

# What has changed
Changed OffsetDateTime field type to String




# How to test?
build local image of this branch and run with docker dev
run acceptance tests and check date formats in event_payload field in events table




# Links
[Trello](https://trello.com/c/eF4c7Q7e/979-dates-in-case-event-payload-json-not-being-stored-correctly-3)




# Screenshots (if appropriate):
